pkg/drivers/kic/types.go

```diff
@@ -24,9 +24,9 @@ import (
 
 const (
 	// Version is the current version of kic
-	Version = "v0.0.12-snapshot"
+	Version = "v0.0.12-snapshot2"
```

Comment:
What's with the snapshot name? Why not versions?

Reply:
This is before we release.

Reply:
so basically snapshot means current HEAD changes that has not been released to people yet
we can potentially use a different repo too but this pattern seems to straighforward

Reply:
the idea is to one day automate this process using this pattern https://github.com/kubernetes/minikube/issues/9072

Reply:
Probably need some additional identifier too, to distinguish between different images.
Could either use commit (from "git describe") or a date (like KIND is doing) perhaps ?

v1.12.3-153-g5dc6c83
v20200817-222e7545